src/database/Matrimonial/migrations/V004_UserProfile_Delete_Update_Procedues.sql

```diff
@@ -0,0 +1,150 @@
+DELIMITER //
+
+CREATE PROCEDURE CreateUserProfile(
+	
+    IN p_UserId INT,
+    IN p_Nationality VARCHAR(255),
+    IN p_Religion VARCHAR(255),
+    IN p_Height VARCHAR(255),
+    IN p_Marital_Status VARCHAR(255),
+    IN p_Bio VARCHAR(255),
+    OUT p_UserProfileId INT,
+    OUT p_Errors VARCHAR(255)
+	
+)
+BEGIN
+	-- Error table
+	SET @ErrorList = '[]';
+    
+    -- add validation
+    
+    IF NOT EXISTS ( SELECT UserId FROM Users WHERE UserId = p_UserId) THEN
+    SET @ErrorList = JSON_ARRAY_APPEND(@ErrorList, '$', 'User Id does not exist.');
+    END IF;
+    
+    IF p_Nationality IS NULL OR p_Nationality = '' THEN
+    SET @ErrorList = JSON_ARRAY_APPEND(@ErrorList, '$', 'Nationality can not be null or empty');
+    END IF;
+    
+    IF p_Religion IS NULL OR p_Religion = '' THEN
+    SET @ErrorList = JSON_ARRAY_APPEND(@ErrorList, '$', 'Religion can not be null or empty');
+    END IF;
+    
+    IF p_Height IS NULL OR p_Height = '' THEN
```

Comment:
We can do a validation using regex to see if this is a propre value for height

![image](https://github.com/user-attachments/assets/f241b67a-6c74-4474-9f58-cb2ff751d67c)


regex value     /^(3-7)'(?:\s*(?:1[01]|0-9)(''|"))?$/